static/schemas/2.0/relations.2.0.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "Relations.2.0",
+    "definitions": {
+        "Relation": {
+            "description": "A key value set of relations",
+            "type": "object",
+            "properties": {
+                "contains": {
+                    "description": "an array of nodes contained by the rack",
+                    "type": "array"
+                },
+                "containedBy": {
+                    "description": "the rack containing the node",
+                    "type": "array"
+                }
+            },
+            "anyOf": [
```

Comment:
Whether user can input both `containedBy` and `contains`? If not, need change `anyOf` to `oneOf`.